crates/walking_engine/src/step_plan.rs

```diff
@@ -42,14 +42,20 @@ impl StepPlan {
             + travel_weighting(
                 swing_foot_travel,
                 turn_travel,
-                parameters.base.foot_lift_apex_increase,
+                parameters
+                    .base
+                    .foot_lift_apex_increase
+                    .div_or_zero(&context.step_planner_parameters.max_step_size),
```

Comment:
in combination with the implementation with `travel_weighting` using `norm` this might be incorrect. We probably should discuss in person